From 1fba2e4bd374035fc884edc9c929a04d48bbf6a6 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 14 Aug 2009 16:16:48 +0200
Subject: [PATCH 08/33] add another version of a SonyEricsson K800i as music player

Added an adapted version of a patch from fd.o#11835, contributed
by Per Thomas Jahr <perja@ifi.uio.no>, to add another version of
a SonyEricsson K800i as USB music player.
---
 .../10freedesktop/10-usb-music-players.fdi         |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index b92192f..cc26e83 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -736,9 +736,10 @@
 
 	  <!-- K800i mobile phone -->
 	  <!-- media files go in mp3/$ARTIST/$ALBUM/$TRACKNAME -->
-	  <match key="@storage.originating_device:usb.product_id" int="0xd039">
+	  <match key="@storage.originating_device:usb.product_id" int_outof="0xd039;0xe039">
 	    <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
             <append key="portable_audio_player.output_formats" type="strlist">audio/3gpp</append>
+            <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
             <append key="portable_audio_player.output_formats" type="strlist">audio/mp4</append>
             <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
             <append key="portable_audio_player.output_formats" type="strlist">audio/vnd.rn-realaudio</append>
@@ -748,6 +749,7 @@
 	    <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
 	    <append key="portable_audio_player.input_formats" type="strlist">audio/x-amr</append>
+	    <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
 	  </match>  	
 
 	  <!-- W950i mobile phone -->
-- 
1.7.1

